server/src/main/java/org/apache/druid/server/coordinator/duty/KillUnusedSegments.java

```diff
@@ -191,38 +207,43 @@ private int killUnusedSegments(
     if (0 < availableKillTaskSlots && !CollectionUtils.isNullOrEmpty(dataSourcesToKill)) {
       for (String dataSource : dataSourcesToKill) {
         if (submittedTasks >= availableKillTaskSlots) {
-          log.debug(StringUtils.format(
+          log.info(StringUtils.format(
```

Comment:
This log I think can get chatty, maybe consider leaving as debug.

Reply:
Yeah, good point, I will move this to debug. Generally while testing this feature, I found it difficult to see what was happening